tests/unsync.rs

```diff
@@ -0,0 +1,72 @@
+#![cfg(feature = "use_std")]
```

Comment:
Are you sure? I actually added this to resolve a CI failure. `cargo test --no-default-features` fails without it.